### PR TITLE
create browse method

### PR DESCRIPTION
**Pull request by maelle**
to address #161

### CODE REVIEW

**Comment by maelle** [comment]
:wave: @gaborcsardi, does this look ok?

File: R/check-class.R [hunk 1]
```diff
@@ -68,6 +68,9 @@ rhub_check <- R6Class(
     print = function(...)
       check_print(self, private, ...),
 
+    browse = function(which = NULL)
+      check_web(self, private, which),
+    
     web = function(which = NULL)
       check_web(self, private, which),
```
Comment (gaborcsardi) [inline R/check-class.R]: Can `browse` call `web`?

Comment (maelle) [reply R/check-class.R]: How? I tried

```r
browse = function(which = NULL)
      web(which)
```

but then I get the error `Error in web(which) : could not find function "web"` when I try using the `browse` method.

Comment (maelle) [reply R/check-class.R]: maybe I can just make it `self$web(which)` :woman_facepalming:

Comment (gaborcsardi) [reply R/check-class.R]: `self$web(which)`

**Comment by gaborcsardi** [comment]
Yes, thanks!

**Pull request merged**